guacamole/src/main/frontend/src/translations/fr.json

```diff
@@ -4,14 +4,20 @@
 
     "APP" : {
 
+        "NAME"    : "Apache Guacamole",
+        "VERSION" : "${project.version}",
```

Comment:
These lines don't need to be included - they don't actually change/override anything, and the English translation ones will be used by default.

Reply:
Removed